packages/browser/README.md

```diff
@@ -0,0 +1,170 @@
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+}
+
+// Initialize the client
+const client = BacktraceClient.builder(options).build();
+
+// When an error occurs
+client.send(new Error("Something broke!"));
+
+```
+## Features
+
+### File Attachments
+***
+Backtrace has several ways in which files can be attached: when initalizing the BacktraceClient, updating the BacktraceClient, or  dynamically for specific reports.  When specifing attachments to the BacktraceClient, all files will be uploaded with each report.
+
+```js
+// Import the attachment types from @backtrace/browser with your favoriate package manager.
+import { BacktraceUint8ArrayAttachment, BacktraceStringAttachment  } from "@backtrace/browser";
+
+// BacktraceStringAttachment should be used for text object like a log file for instance
+var attachment1 = new BacktraceStringAttachment("logfile.txt", "This is the start of my log")
+
+// BacktraceUint8ArrayAttachment should be used for binary files
+var attachment2 = new BacktraceUint8ArrayAttachment("output.dmp", new Uint8Array(2));
+
+// Setup array of files to attach
+const attachments = [attachment1];
+
+// BacktraceClientOptions
+const options = {
+    // Name of the website/application
+    name: "MyWebPage",
+    // Version of the website
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+    // Attach the files to all reports
+    attachments: attachments
+}
+
+// Later decide to add an attachment to all reports
+client.attachments.push(attachment2)
+
+// After catching an exception and generating a report
+try {
+    throw new Error("Caught exception!")
+} catch (error) {
+    const report = new BacktraceReport(error);
+    const attachment3 = new BacktraceStringAttachment("CaughtErrorLog", "some error logging data here");
+    report.attachments.push(attachment3);
+    client.send(report);
+}
+```
+
+### Metrics
+Backtrace Metrics tracks unique application launches and unique identifiers (default: guid) and submits them to Backtrace.  This allows for overview in the Backtrace web console of how many errors occur compared to all active users for a given platform, version, and more to be seen.
+
+Backtrace Metrics is enabled by default.  The following options can be passed to the `metrics` [BacktraceClientOptions](#backtraceclientoptions) item.
+
+| Option Name | Type | Description | Default |
+|-|-|-|-|
+|`metricsSubmissionUrl`|String|Metrics server hostname|`https://events.backtrace.io`|
+|`enable`|Boolean|Determines if the metrics support is enabled|`true`|
+|`autoSendInterval`|Integer|Indicates how often crash free metrics are sent to Backtrace. The interval is a value in ms. The default is 30 minutes|`1800000`|
+|`size`|Integer|Indicates how many events the metrics storage can store before auto submission|`50`|
+
+### Breadcrumbs
+Feature in developement
+
+### Callback Methods
```

Comment:
I would tentatively assert:
Callback methods would be part of Documentation.
"Features" would be info relating to how to support a Backtrace system/UI feature.
"Documentation" would be info on just the SDK - anything mechanical that happens prior to the report being sent.